File: app/src/main/java/dk/lndesign/relay/irc/IRCReturnCode.java
package dk.lndesign.relay.irc;

/**
 * List of IRC return codes.
 * {@see http://www.networksorcery.com/enp/protocol/irc.htm}
 * @author Lars Nielsen <lars@lndesign.dk>
 */
public class IRCReturnCode {

    public static final String RPL_MYINFO = "004";
    public static final String RPL_NAMREPLY = "353";
    public static final String RPL_ENDOFNAMES = "366";
    public static final String RPL_MOTDSTART = "375";
    public static final String RPL_ENDOFMOTD = "376";
    public static final String RPL_MOTD = "372";

    public static final String ERR_NOSUCHNICK = "401";
    public static final String ERR_NOSUCHSERVER = "402";
    public static final String ERR_NOSUCHCHANNEL = "403";
    public static final String ERR_NICKNAMEINUSE = "433";
}

File: app/src/main/java/dk/lndesign/relay/irc/IRCReturnCode.kt
package dk.lndesign.relay.irc

/**
 * List of IRC return codes.
 * {@see http://www.networksorcery.com/enp/protocol/irc.htm}
 */
object IRCReturnCode {

    val RPL_MYINFO = "004"
    val RPL_NAMREPLY = "353"
    val RPL_ENDOFNAMES = "366"
    val RPL_MOTDSTART = "375"
    val RPL_ENDOFMOTD = "376"
    val RPL_MOTD = "372"

    val ERR_NOSUCHNICK = "401"
    val ERR_NOSUCHSERVER = "402"
    val ERR_NOSUCHCHANNEL = "403"
    val ERR_NICKNAMEINUSE = "433"
}
